### PR TITLE
add windows related information

### PR DESCRIPTION
**Pull request by fmeynard**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,17 @@ make configure
 make all
 ```
 
+#### Windows
+
+* Install [WSL2](https://docs.microsoft.com/en-us/windows/wsl/wsl2-install)
+* Enable [Docker support  for WSL2](https://docs.docker.com/docker-for-windows/wsl-tech-preview/)
+* Checkout project directly within WSL, ( Using a project directory from windows directly will cause massive performances issues and watchers ( i.e : yarn encore ) to not work at all.
```
Comment (mRoca) [inline README.md]: don't forget to close your "(" ;-)

File: README.md [hunk 2]
```diff
@@ -38,6 +38,17 @@ make configure
...
+* Run Linux steps
```
Comment (mRoca) [inline README.md]: Run `Linux` build steps

File: README.md [hunk 3]
```diff
@@ -38,6 +38,17 @@ make configure
...
+
+_Depending on Linux distribution used by WSL you will have to install some packages like make, npm etc._
```
Comment (mRoca) [inline README.md]: Why npm ?

File: README.md [hunk 4]
```diff
@@ -38,6 +38,17 @@ make configure
 make all
 ```
 
+#### Windows
+
+* Install [WSL2](https://docs.microsoft.com/en-us/windows/wsl/wsl2-install)
+* Enable [Docker support  for WSL2](https://docs.docker.com/docker-for-windows/wsl-tech-preview/)
```
Comment (mRoca) [inline README.md]: don't you need to add docker-compose manually ?

Comment (fmeynard) [reply README.md]: Nop, just enabling the WSL2 support in the Docker Desktop applications and thats fine

**Pull request merged**